profiler/profiler.heapwalker/src/org/netbeans/modules/profiler/heapwalk/ClassesListController.java

```diff
@@ -164,7 +164,7 @@ public Object[][] getData(String[] filterStrings, int filterType, boolean showZe
                     else data[i][4] = (retainedSizeByClass > 0 ? "+" : "") + numberFormat.format(retainedSizeByClass); // NOI18N 
                 }
             } else {
-                data[i][1] = new Double((double) instancesCount /
+                data[i][1] = Double.valueOf((double) instancesCount /
```

Comment:
This `Double#valueOf` wrap can be dropped. The type of the inner expression is `double`, autoboxing should do the rest.

Reply:
Agreed..